README.md

```diff
@@ -157,7 +157,7 @@ first = "Tom" last = "Preston-Werner" # INVALID
 Keys
 ----
 
-A key may be either bare, quoted, or dotted.
+A key is treated as a string and may be either bare, quoted, or dotted.
```

Comment:
I *really* don't want us to be referencing a value type, that we describe as a much later in the specification to describe keys.

Further, I don't think this addresses the "is bare same as quoted" concern from the original issue.

Reply:
Good point. I can't say that I didn't think that talking about "strings" in this way may be confusing. What I was trying to convey was, if you've got a key in double-quotes, resolve it like you would a string value, in order to get the key's proper name. On reflection, that's not so obvious.